README.md

```diff
@@ -38,7 +38,7 @@ provider "azurerm" {
 }
 
 module "policy" {
-  source              = "../../"
+  source              = "git::https://github.com/OT-terraform-azure-modules/terraform-azure-policies.git"
```

Comment:
The source URL should not be GITHUB URL but Terraform Registry URL.

source  = "OT-terraform-azure-modules/policies/azure"
version = "0.0.2"